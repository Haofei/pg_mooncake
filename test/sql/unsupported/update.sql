CREATE TABLE t (a int, b int) USING columnstore;
UPDATE t SET (a, b) = (SELECT 1, 2);
UPDATE t SET (b, a) = (SELECT 1, 2);
DROP TABLE t;
