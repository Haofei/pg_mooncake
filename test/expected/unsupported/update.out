CREATE TABLE t (a int, b int) USING columnstore;
UPDATE t SET (a, b) = (SELECT 1, 2);
ERROR:  DuckDB does not support UPDATE with multi-column assignment
UPDATE t SET (b, a) = (SELECT 1, 2);
ERROR:  DuckDB does not support UPDATE with multi-column assignment
DROP TABLE t;
